cmd/lifecycle/exporter.go

```diff
@@ -64,12 +64,12 @@ func (e *exportCmd) DefineFlags() {
 	cli.FlagProcessType(&e.DefaultProcessType)
 	cli.FlagProjectMetadataPath(&e.ProjectMetadataPath)
 	cli.FlagReportPath(&e.ReportPath)
-	cli.FlagRunImage(&e.RunImageRef)
+	cli.FlagRunImage(&e.RunImageRef) // Note: this flag isn't valid on Platform 0.7 and later, and any provided value is ignored
```

Comment:
```suggestion
	cli.FlagRunImage(&e.RunImageRef) // Note: this flag isn't valid on Platform 0.7 and later
```
(randomly found when reviewing the code) Actually, I don't think we even ignore the provided value :/ we should probably change this, especially when we formally deprecate platforms older than 0.7

Reply:
wait what does that mean,  that it's not valid but "we don't even ignore the provided value" -- what do we do with it?